packages/framework-provider-kubernetes/src/services/redis-adapter.ts

```diff
@@ -0,0 +1,51 @@
+import { Logger } from 'framework-types/dist'
+import fetch from 'node-fetch'
+const redis = require("redis");
+
+export class RedisAdapter {
+
+    constructor(readonly daprUrl: string, readonly redisUrl: string) {}
+
+    public static build() : RedisAdapter {
+        const redisPwd: string = process.env['DB_PASSWORD'] || 'password'
+        const redisUrl: string = `redis://redis-master:6379?password=${redisPwd}`
```

Comment:
Should this be hardcoded? Maybe we have to pass the URL with env variable?

Reply:
Yes, the host could be passed as an env variable

Reply:
upss!! my fault! totally agree it can be also passed as an ENV var. @davidverdu let's suppose that we have a `DB_HOST` passing the host + port and I'll add it in my next PR, so the line would be: `redis://${dbHost}?....`